client/api.go

```diff
@@ -35,119 +37,99 @@ import (
-	Cards *card.Client
-	// Subscriptions is the client used to invoke /subscriptions APIs.
-	// For more details see https://stripe.com/docs/api#subscriptions.
-	Subscriptions *sub.Client
-	// SubscriptionItems is the client used to invoke subscription's items-related APIs.
-	// For more details see https://stripe.com/docs/api#subscription_items.
-	SubscriptionItems *subitem.Client
-	// Plans is the client used to invoke /plans APIs.
-	// For more details see https://stripe.com/docs/api#plans.
-	Plans *plan.Client
+	// CountrySpec is the client used to invoke /country_specs APIs.
+	CountrySpec *countryspec.Client
 	// Coupons is the client used to invoke /coupons APIs.
-	// For more details see https://stripe.com/docs/api#coupons.
 	Coupons *coupon.Client
-	// Discounts is the client used to invoke discount-related APIs.
-	// For mode details see https://stripe.com/docs/api#discounts.
+	// Customers is the client used to invoke /customers APIs.
+	Customers *customer.Client
+	// Discounts is the client used to invoke discount related APIs.
 	Discounts *discount.Client
+	// Disputes is the client used to invoke /disputes APIs.
+	Disputes *dispute.Client
+	// EphemeralKeys is the client used to invoke /ephemeral_keys APIs.
+	EphemeralKeys *ephemeralkey.Client
+	// Events is the client used to invoke /events APIs.
+	Events *event.Client
+	// ExchangeRates is the client used to invoke /exchange_rates APIs.
+	ExchangeRates *exchangerate.Client
+	// Fees is the client used to invoke /application_fees APIs.
+	Fees *fee.Client
+	// FeeRefunds is the client used to invoke /application_fees/refunds APIs.
+	FeeRefunds *feerefund.Client
+	// FileUploads is the client used to invoke the /files APIs.
+	FileUploads *fileupload.Client
 	// Invoices is the client used to invoke /invoices APIs.
-	// For more details see https://stripe.com/docs/api#invoices.
 	Invoices *invoice.Client
 	// InvoiceItems is the client used to invoke /invoiceitems APIs.
-	// For more details see https://stripe.com/docs/api#invoiceitems.
 	InvoiceItems *invoiceitem.Client
-	// LoginLinks is the client used to invoke /v1/accounts/<account_id>/login_links APIs.
-	// For more details see https://stripe.com/docs/api#login_link_object.
+	// IssuerFraudRecords is the client used to invoke /issuer_fraud_records APIs.
+	IssuerFraudRecords *issuerfraudrecord.Client
+	// LoginLinks is the client used to invoke login link related APIs.
 	LoginLinks *loginlink.Client
-	// Disputes is the client used to invoke dispute-related APIs.
-	// For more details see https://stripe.com/docs/api#disputes.
-	Disputes *dispute.Client
-	// Transfers is the client used to invoke /transfers APIs.
-	// For more details see https://stripe.com/docs/api#transfers.
-	Transfers *transfer.Client
+	// Orders is the client used to invoke /orders APIs.
+	Orders *order.Client
+	// OrderReturns is the client used to invoke /order_returns APIs.
+	OrderReturns *orderreturn.Client
```

Comment:
Was not sure if `Orders` was before `OrderReturns` (as in whether the case matters in alphabetical ordering). It seemed easier/cleaner this way though. (true for other packages like sub below)

Reply:
Technically I think `OrderReturns` would come first. I don't think it matters that much though — like you said, this will be more of a sane ordering for users reading through this list.